quiz.html

```diff
@@ -125,14 +126,85 @@ <h4 class="quiz-header" id="quiz-header">
           </div>
         </div>
       </div>
+      <div class="quiz-right-bottom hidden" id="quiz-right-bottom-3">
+
+        <table>
+          <tr>
+            <th>RANK</th>
+            <th>NAME</th>
+            <th>SCORE</th>
+          </tr>
+          <tr>
+            <td>1st</td>
+            <td id="scoreName1"></td>
+            <td id="scoreScore1"></td>
+          </tr>
+          <tr>
+            <td>2nd</td>
+            <td id="scoreName2"></td>
+            <td id="scoreScore2"></td>
+          </tr>
+          <tr>
+            <td>3rd</td>
+            <td id="scoreName3"></td>
+            <td id="scoreScore3"></td>
+          </tr>
+          <tr>
+            <td>4th</td>
+            <td id="scoreName4"></td>
+            <td id="scoreScore4"></td>
+          </tr>
+          <tr>
+            <td>5th</td>
+            <td id="scoreName5"></td>
+            <td id="scoreScore5"></td>
+          </tr>
+          <tr>
+            <td>6th</td>
+            <td id="scoreName6"></td>
+            <td id="scoreScore6"></td>
+          </tr>
+          <tr>
+            <td>7th</td>
+            <td id="scoreName7"></td>
+            <td id="scoreScore7"></td>
+          </tr>
+          <tr>
+            <td>8th</td>
+            <td id="scoreName8"></td>
+            <td id="scoreScore8"></td>
+          </tr>
+          <tr>
+            <td>9th</td>
+            <td id="scoreName9"></td>
+            <td id="scoreScore9"></td>
+          </tr>
+          <tr>
+            <td>10th</td>
+            <td id="scoreName10"></td>
+            <td id="scoreScore10"></td>
+          </tr>
+        </table>
+        <div class='quiz-button quiz-small' onClick="copyScore()">Copy your score to your clipboard</div>
+        <div class='quiz-button quiz-small' onClick=history.go(0)>Another go?</div>
+      </div>
+      <div class="quiz-right-bottom hidden" id="quiz-right-bottom-4">        
+        <div class="enterScore">
+          Enter your initials:<br>
+          <input id="scoreInput" type="text" required="required" minlength="3" maxlength="3"></input><br>
+          <input id="scoreButton" type="submit" onclick="enterScore()"></input>
+        </div>
+      </div>
     </div>
   </div>
 
+  <script src="https://www.gstatic.com/firebasejs/8.10.0/firebase-app.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/8.10.0/firebase-firestore.js"></script>
   <script src="js/scripts.min.js"></script>
   <script src="js/main.min.js"></script>
+  <script src="js/data.js"></script>
   <script src="js/quiz.js"></script>
 
-
 </body>
 
 </html>
```

Comment:
```suggestion
</html>

```
we need a newline at the end of the file